Tests/js/views/actions/assets/ez-translateactionview-tests.js

```diff
@@ -0,0 +1,334 @@
+                content: this.contentMock,
+                location: this.locationMock
+            });
+            this.view.render();
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        'Should render links': function () {
+            var container = Y.one('.container'),
+                translations = container.all('.ez-contenttranslation');
+
+            Y.each(translations, function (translation) {
+                Assert.isNotNull(
+                    translation.one('.ez-contenttranslation-view-link'),
+                    'Should contain link to view location'
+                );
+                Assert.isNotNull(
+                    translation.one('.ez-contenttranslation-edit-link'),
+                    'Should contain link to edit content'
+                );
+            });
+        },
+    });
+
+    hideTest = new Y.Test.Case({
+        name: 'eZ Translate Action View hide test',
+
+        setUp: function () {
+            this.contentMock = new Mock();
+            this.locationMock = new Mock();
+            this.versionMock = new Mock();
+            this.translationsList = ['eng-GB', 'pol-PL'];
+
+            Mock.expect(this.contentMock, {
+                method: 'get',
+                args: ['currentVersion'],
+                returns: this.versionMock
+            });
+            Mock.expect(this.versionMock, {
+                method: 'getTranslationsList',
+                returns: this.translationsList
+            });
+            Mock.expect(this.locationMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+            Mock.expect(this.contentMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            this.view = new Y.eZ.TranslateActionView({
+                container: '.container',
+                actionId: 'translate',
+                label: 'Translations',
+                disabled: false,
+                content: this.contentMock,
+                location: this.locationMock
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should hide the view when clicking outside of the view": function () {
+            this.view.render();
+            this.view.set('expanded', true);
+            Y.one('.click-somewhere').simulate('click');
+
+            Assert.isFalse(this.view.get('expanded'), "The view should be hidden");
+        },
+
+        "Should leave the view when clicking outside of the view": function () {
+            this.view.render();
+            this.view.set('expanded', true);
+            this.view.set('expanded', false);
+            Y.one('.click-somewhere').simulate('click');
+
+            Assert.isFalse(this.view.get('expanded'), "The view should be hidden");
+        }
+    });
+
+    hintTest = new Y.Test.Case({
```

Comment:
Now testing the hint by comparing attribute with expected value. BTW found out that test fails because of keeping reference to translations, so I had to clone the translations array and make a splice on it: https://github.com/ezsystems/PlatformUIBundle/pull/277/files#diff-659f9de6ec2e07d2db4ea7912dc5e01cR104